### PR TITLE
Docs: rename the select-from-mutation-fragments page name

### LINKED ISSUE #17324: docs: Issue in page SELECT * FROM MUTATION_FRAGMENTS() Statement - page seems out of place
I would like to report an issue on page https://opensource.docs.scylladb.com/branch-5.4/operating-scylla/admin-tools/select-from-mutation-fragments

### Problem

The header looks (literally) out of line in the tool index
image

###  Suggest a fix
Renaming the page to something like "Reading mutations data" would look better.

### PR DESCRIPTION
**Pull request by tzach**
Fix #17324 

### before
image

### After
image

### CODE REVIEW

File: docs/operating-scylla/admin-tools/select-from-mutation-fragments.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ============================================
-SELECT * FROM MUTATION_FRAGMENTS() Statement
+Reading mutations fragments
 ============================================
```
Comment (denesb) [inline docs/operating-scylla/admin-tools/select-from-mutation-fragments.rst:3]: You possibly have to remove some of the `===`. I don't know if it matters or not, @annastuchlik?

**Review by denesb — APPROVED** [review]

**Comment by tzach** [comment]
> @tzach Please update both descriptions from"Reading mutations fragments" to "Reading mutation fragments" (no plural).

Is it plural? I followed the table name convention.

**Comment by tzach** [comment]
> It should follow grammar and be consistent with "MUTATION_FRAGMENTS"

This is not about grammer, its about the content of the table.
Are the fragments belong to one mutation or to many muration*s*?

**Comment by denesb** [comment]
> Are the fragments belong to one mutation or to many muration_s_?

Yes. Depends on how many mutations (partitions) you query. There are multiple fragments (at least 2) for each mutation (partition).

**Comment by denesb** [comment]
But note, that the name of this modifier (`MUTATION_FRAGMENTS`), is derived from the internal C++ class `mutation_fragment`. Use din plural, this is "mutation fragments". The modifier was intended to imply that the content of the query will presented in terms of `mutation_fragment`:s (there are always at least 2 of these, in any query).

**Comment by denesb** [comment]
All that many words is to say that I think "mutation fragments" is the correct and better sounding variant.

**Comment by tzach** [comment]
Sorry, forgot to push. @annastuchlik please review now.

**Pull request closed without merging**